### PR TITLE
Implement script to add new tables to demo db for the next release

### PR DESCRIPTION
**Pull request by cw75**
## Describe your changes and why you are making these changes
Tested that it works with both fresh install and package upgrade.

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/python/bin/aqueduct [hunk 1]
```diff
@@ -246,8 +263,6 @@ def update():
             with open(os.path.join(server_directory, "db/demo.db"), "wb") as f:
                 f.write(requests.get(os.path.join(s3_server_prefix, "db/demo.db")).content)
 
-            update_server_version()
```
Comment (cw75) [inline src/python/bin/aqueduct:249]: realized we can consolidate this to line 282 below.

File: src/resources/demo_db/sqlite/v0.1/create_tables.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from os.path import join
```
Comment (cw75) [inline src/resources/demo_db/sqlite/v0.1/create_tables.py:1]: This script is here for reference; It will be fetched from S3 during the update process.

Comment (likawind) [reply src/resources/demo_db/sqlite/v0.1/create_tables.py:1]: In future, should each version with new table contains a `create_tables.py` in that version's directory?

File: src/python/bin/aqueduct [hunk 3]
```diff
@@ -214,6 +218,19 @@ def update_server_version():
         ]
     )
 
+    if parse_version(current_version) < parse_version("0.1") and parse_version(package_version) >= parse_version("0.1"):
```
Comment (likawind) [inline src/python/bin/aqueduct:221]: Probably fine for now, but we like need it to be a helper so that it's easier to add new tables in future.

**Review by likawind — APPROVED** [review]

File: src/python/bin/aqueduct (hunk 3 above)
Comment (cw75) [reply src/python/bin/aqueduct:221]: Yup when we have many of these, we can package each of them into a helper function called from this main function. I'll add a comment as a reminder.

**Pull request merged**